python/docs/intro.rst

```diff
@@ -88,6 +88,10 @@ This enables quick and easy mathematical operations on the data you care about.
 
 Please refer to :py:class:`dlisio.plumbing.Channel` and :py:class:`dlisio.plumbing.Frame`
 
+Strings and encodings
+=====================
```

Comment:
Maybe use different type of header? It's now a main section along with About and DLIS.

Reply:
That was intentional, it's a quite global topic. Are there anywhere else it rather belongs?

Reply:
No, I am ok with having it here, just with '------' section, so it's on the same level as Curves and Physical and logical files

Reply:
But it really hasn't got anything to do with DLIS, right? And presumably there will be more, similar sections later.

Reply:
I wouldn't say that it has nothing to do with DLIS, but OK, I don't mind it that much :)

Reply:
Well, it deals with _violations_ of DLIS, but it's really not on the levels of logs and curves and what-not.